uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -509,7 +507,31 @@ define([
         }
       });
 
-      return calendar;
+      return indexedByMonth;
+    }
+
+    /**
+     * Show month loader for all months initially
+     * then hide each loader on the interval of an offset value
+     */
+    function showMonthLoader() {
+      var offset = 0;
+
+      vm.months.forEach(function (month, index) {
+        //show loader
+        vm.loading.months[index] = (index !== vm.months.indexOf(vm.selectedMonths[0]));
```

Comment:
Here I am setting loader to true for all months expect for the current month which is selected by default. Ss this function is only called initially, there will a single items in `selectedMonths`, so I am not setting loader to true for current month(`vm.months.indexOf(vm.selectedMonths[0])` is the index of current month)